pkg/router/functionServiceMap.go

```diff
@@ -42,6 +42,8 @@ type (
 )
 
 func makeFunctionServiceMap(logger *zap.Logger, expiry time.Duration) *functionServiceMap {
+	logger.Info("creating functionServiceMap")
```

Comment:
I don't understand the purpose of this log line?

What is it going to answer? You already log in `Start` and this function is only called during `Start`.